main.go

```diff
@@ -210,11 +220,58 @@ func main() {
 	}
 }
 
+func prepareFilter(filters []string) (tracee.Filter, error) {
+
+	uids := []uint32{}
+
+	filterHelp := "\n--filter allows you to specify values to match on for fields of traced events.\n"
+	filterHelp += "The following options are currently supported:\n"
+	filterHelp += "uid: only trace processes or containers with specified uid(s).\n"
+	filterHelp += "\t--filter uid=0                                                | only trace events from uid 0 or uid 1000\n"
+	filterHelp += "\t--filter uid=0,1000                                           | only trace events from uid 0\n"
```

Comment:
Comments of the above two lines mixed?

Reply:
Ah thought I fixed this, Idk how I lost that change. Pushing now.